react/react1/week2/TodoList/src/components/TodoHeader.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function TodoHeader() {
+  return <h1>Todo List</h1>;
+}
+
```

Comment:
To make this component truly dynamic, it could accept at least the ``title`` as prop and render that 💡